ote/src/cljs/ote/views/admin/detected_changes.cljs

```diff
@@ -33,22 +49,40 @@
       [ui/table-body {:display-row-checkbox false}
        (doall
          (for [s services]
-           ^{:key (str "link_" s)}
+           ^{:key (:servce-id s)}
```

Comment:
That key looks funky, should it be :service-id ?

Reply:
:service-id is only a keyword. (:service-id s) is shorthand for (get s :service-id)